nautobot_ssot/jobs/examples.py

```diff
@@ -107,19 +147,48 @@ class PrefixModel(NautobotModel):
     _modelname = "prefix"
     _identifiers = ("network", "prefix_length", "tenant__name")
     # To keep this example simple, we don't include **all** attributes of a Prefix here. But you could!
-    _attributes = ("description", "status__name")
+    _attributes = ("description", "namespace__name", "status__name", "locations")
 
     # Data type declarations for all identifiers and attributes
     network: str
+    namespace__name: str
     prefix_length: int
     tenant__name: Optional[str]
     status__name: str
     description: str
 
+    locations: Optional[List[LocationDict]] = []
```

Comment:
```suggestion
    locations: List[LocationDict] = []
```

I think if you default to an empty list it isn't necessary to accept `None` as well, right?

Reply:
Fair point, it does seem to be a [] by default in the API response so will remove Optional.